docs/topics/api/scanners.rst

```diff
@@ -0,0 +1,22 @@
+============
+Scanners
+============
+
+.. note::
+
+    These APIs are subject to change at any time and are for internal use only.
+
+
+---------------------
+Scanner Results
+---------------------
+
+.. _scanner-results:
+
+This endpoint returns a list of labelled scanner results.
+
+.. http:get:: /api/v4/scanner/results/
+
+    :>json int id: The scanner result ID.
+    :>json string scanner: The scanner name.
+    :>json object results: The scanner (raw) results.
```

Comment:
It would be good if we could document a breakdown of the object structure, though I guess that's a) not determined currently and b) could change from scanner to scanner (and c) maybe something we can't expose publicly)

Reply:
it is definitely (b) and (c) :wink:

Reply:
I wonder if we have some potential structures documented in the scanners repositories and link there with some "this link may require specific permissions" notice or so? That way we'd document the structures of each scanner individually (doesn't have to be in-depth documentation just now) and link to specific scanners individually.

Reply:
We discussed about this and decided not to do it :)

Reply:
FWIW, each scanner and/or repo has its own docs and things are documented but we won't link to the different scanners/repos.